### ISSUE TITLE
Placeholder text is not visible enough in search bar. 

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#199]
Placeholder text is not visible enough in search bar.

**Comment by EdwinGuzman** [comment]
@ricardogalvez , by the way, I had to add specific CSS rules for the placeholder for different browsers. Here's the appropriate [PR in the UI](https://github.com/NYPL-discovery/discovery-front-end/pull/660). Sorry, I didn't know this was a ticket in the Design-Toolkit or else I would've told you sooner.
